torch/_dynamo/variables/misc.py

```diff
@@ -297,6 +297,11 @@ def call_method(
                 tx.symbolic_torch_function_state.torch_function_subclass_enabled = (
                     tx_old
                 )
+        elif isinstance(inner_fn, types.MethodDescriptorType):
```

Comment:
Should we limit this to `torch.Tensor` method descriptors? (I think I did that elsewhere)

Reply:
Yep added an extra `inner_fn in trace_rules.get_tensor_method()` condition: https://github.com/pytorch/pytorch/blob/295162ec3abb2a58ab6a54af15196ea6fadf4852/torch/_dynamo/trace_rules.py#L2914-L2936